src/sentry/seer/endpoints/organization_seer_explorer_chat.py

```diff
@@ -184,8 +191,9 @@ def post(
         query = validated_data["query"]
         insert_index = validated_data.get("insert_index")
         message_timestamp = validated_data.get("message_timestamp")
+        on_page_context = validated_data.get("on_page_context")
 
         response_data = _call_seer_explorer_chat(
```

Comment:
nit: this argument list is getting long, it could be worth refactoring `_call_seer_explorer_chat` to force named keyword arguments (prepend * to argument list), as calls like:
```
mock_call_seer_chat.assert_called_once_with(
            self.organization, None, "What is this error about?", None, None, None
        )
``` 
start to become hard to understand.